packages/components/src/mobile/bottom-sheet/range-cell.native.js

```diff
@@ -154,20 +153,18 @@ class BottomSheetRangeCell extends Component {
 		);
 
 		const defaultSliderStyle = getStylesFromColorScheme( styles.sliderTextInput, styles.sliderDarkTextInput );
-		const resetButtonText = Platform.OS === 'ios' ? __( 'Reset' ) : __( 'RESET' );
-		const resetButton = { title: resetButtonText, handler: this.handleReset };
```

Comment:
Do we still need this? https://github.com/WordPress/gutenberg/blob/dc229b4f937761dd818b46078049ae863d6d718b/packages/components/src/mobile/bottom-sheet/range-cell.native.js#L71

Reply:
> We do have the leftAlign prop.

I've noticed that, however that little marging was confusing for me. If we can remove it I'm all for that!

> Do we still need this? -> handleReset() {}

We don't need it, I was rushing a bit and I must have missed it.

Reply:
> I've noticed that, however that little margin was confusing for me. If we can remove it I'm all for that!

Double checked this and this margin is being used in an Android variant, so let's keep it as it is for now. If we end up having more options without icons, we'll need to modify the Cell's label styles logic. 👍

Reply:
So I'm going to remove `handleReset` leftovers 👍

Reply:
Removed.